layerindex/context_processors.py

```diff
@@ -1,21 +1,33 @@
 # layerindex-web - custom context processor
 #
-# Copyright (C) 2013 Intel Corporation
+# Copyright (C) 2013, 2018 Intel Corporation
 #
 # Licensed under the MIT license, see COPYING.MIT for details
+#
+# SPDX-License-Identifier: MIT
 
-from layerindex.models import Branch, LayerItem
+from layerindex.models import Branch, LayerItem, SiteNotice
 from django.contrib.sites.models import Site
+from django.db.models import Q
+from datetime import datetime
 
 def layerindex_context(request):
+    import settings
     site = Site.objects.get_current()
     if site and site.name and site.name != 'example.com':
         site_name = site.name
     else:
-        site_name = 'OpenEmbedded metadata index'
+        site_name = 'OpenEmbedded Layer Index'
+    if request.path.startswith('/accounts') or request.path.startswith('/admin/logout'):
+        login_return_url = ''
+    else:
+        login_return_url = request.path
     return {
-        'all_branches': Branch.objects.exclude(name='oe-classic').order_by('sort_priority'),
+        'all_branches': Branch.objects.exclude(comparison=True).exclude(hidden=True).order_by('sort_priority'),
         'unpublished_count': LayerItem.objects.filter(status='N').count(),
-        'oe_classic': Branch.objects.filter(name='oe-classic'),
-        'site_name': site_name
-    }
\ No newline at end of file
+        'site_name': site_name,
+        'rrs_enabled': 'rrs' in settings.INSTALLED_APPS,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)